Samples/Widgets/MultipleWidgets/cpp-win32-packaged/SampleWidgetProviderAppPackage/Package.appxmanifest

```diff
@@ -0,0 +1,124 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:uap3="http://schemas.microsoft.com/appx/manifest/uap/windows10/3"
+  xmlns:com="http://schemas.microsoft.com/appx/manifest/com/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  IgnorableNamespaces="uap uap3 rescap">
+
+	<Identity
+	  Name="SampleWidgetProviderAppPackage"
+	  Publisher="CN=Microsoft Corporation"
+	  Version="1.0.0.0" />
+
+	<Properties>
+		<DisplayName>SampleWidgetProviderAppPackage</DisplayName>
+		<PublisherDisplayName>Microsoft Corporation</PublisherDisplayName>
+		<Logo>Images\StoreLogo.png</Logo>
+	</Properties>
+
+	<Dependencies>
+		<TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.17763.0" MaxVersionTested="10.0.19041.0" />
+		<TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.17763.0" MaxVersionTested="10.0.19041.0" />
+	</Dependencies>
+
+	<Resources>
+		<Resource Language="x-generate"/>
+	</Resources>
+
+	<Applications>
+		<Application Id="App"
+		  Executable="$targetnametoken$.exe"
+		  EntryPoint="$targetentrypoint$">
+			<uap:VisualElements
+			  DisplayName="SampleWidgetProviderAppPackage"
+			  Description="SampleWidgetProviderAppPackage"
+			  BackgroundColor="transparent"
+			  Square150x150Logo="Images\Square150x150Logo.png"
+			  Square44x44Logo="Images\Square44x44Logo.png">
+				<uap:DefaultTile Wide310x150Logo="Images\Wide310x150Logo.png" />
+				<uap:SplashScreen Image="Images\SplashScreen.png" />
+			</uap:VisualElements>
+			<Extensions>
+				<com:Extension Category="windows.comServer">
+					<com:ComServer>
+						<com:ExeServer Executable="SampleWidgetProviderApp\SampleWidgetProviderApp.exe" Arguments="-RegisterProcessAsComServer" DisplayName="SampleWidgetProviderApp">
+							<com:Class Id="101D03A3-6FC8-4887-9B64-310A7B164319" DisplayName="SampleWidgetProvider" />
+						</com:ExeServer>
+					</com:ComServer>
```

Comment:
yup, dll + a surrogate host should be fine as well :)
do you mind sharing your example? I'll add it to the docs

Reply:
Should have a sample up Monday! Will ping you.

Reply:
@riverar have you manage to get that sample together by any chance?:)

Reply:
Rust widget sample https://github.com/microsoft/windows-rs/commit/44313fc2c63f692824214506ed06466bf6a9e3e7 Feel free to hit me up on Teams (rafael@withinwindows.com is in the guest tenant) and we can quickly transform this into a surrogate host sample. (I didn't go down this route for the official sample as it's harder to debug for devs.)

Reply:
@riverar - thanky you! but I was looking specifically for `surrogate host` sample :) I'll try to put it together and ping you in case I encounter blockers:)

Reply:
@beervoley Here's the surrogate host impl, very minor changes https://github.com/riverar/windows-rs/commit/bd3e47e946def205ccee69193a5672188e3d1f76